class-tgm-plugin-activation.php

```diff
@@ -2272,7 +2278,8 @@ protected function get_plugin_status_text( $slug ) {
 			}
 
 			return sprintf(
-				_x( '%1$s, %2$s', '%1$s = install status, %2$s = update status', 'tgmpa' ),
+				/* translators: %1$s: install status, %2$s: update status */
+				__( '%1$s, %2$s', 'tgmpa' ),
 				$install_status,
```

Comment:
In this case, I'd prefer to have both the translators comment and the context. The TGMPA library is included in themes as well as plugins and in the case of wp.org hosted themes, the text-domain is replaced by the theme text-domain.
If, for whatever reason, the theme would have the same generic string in use, things could become very confusing for translators.
For the other strings, I'd think that to be far less likely.